app/views/registration_wizard/shared/_question_page.html.erb

```diff
@@ -0,0 +1,28 @@
+<% content_for :title do %>
+  <%= form.errors.present? ? "Error: " : nil %><%= form.question.label %>
+<% end %>
+
+<% content_for :before_content do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: "Back",
+    href: registration_wizard_show_path(wizard.previous_step_path)
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: form, url: registration_wizard_form_url(form), scope: 'registration_wizard', method: :patch do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <%=
+        render(
+          "registration_wizard/shared/questions/#{form.question.type}",
+          form: f,
+          question: form.question
+        )
+      %>
+
+      <%= f.govuk_submit %>
+    <% end %>
+  </div>
+</div>
```

Comment:
The base question page, includes all the logic required for pretty much all our question pages. It'll acquire more functionality over time but for now this should fit the bill for almost everything.